flent/runners.py

```diff
@@ -680,8 +680,17 @@ def find_binary(self, **args):
             args['socket_timeout'] = ""
 
         if args['test'] in ("TCP_STREAM", "TCP_MAERTS", "omni"):
+            self.ctrl_port = args['netperf_control_port']
```

Comment:
There's no 'netperf_control_port' in args, so this just crashes.

We don't actually configure the control port anywhere, so you'll just have to hard-code it somewhere. Maybe making ssrunner more generic by having a notion of "excluded ports" and passing in 12865 in an argument to the constructor or something?